catalog-info.yaml

```diff
@@ -9,6 +9,6 @@ metadata:
   annotations:
     github.com/project-slug: upbound/upjet
 spec:
-  type: library
+  type: service
```

Comment:
Wasn't this a `service` before we converted it to a `library`?

Reply:
I think service enables the `dependsOn` field, whereas library doesn't. Let's merge and see how it behaves.